doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,661 @@
+  * The :ref:`channel_sounding_ras_initiator` sample demonstrating Channel Sounding by setting up a Channel Sounding Initiator that acts as a Ranging Requestor GATT Client.
+    It includes a basic distance estimation to demonstrate IQ data handling.
+    The accuracy is not representative for Channel Sounding and should be replaced if accuracy is important.
+  * The :ref:`bt_peripheral_with_multiple_identities` sample demonstrating how to use a single physical device to create and manage multiple advertisers, making it appear as multiple distinct devices by assigning each a unique identity.
+  * The :ref:`bt_scanning_while_connecting` sample demonstrating how to establish multiple connections faster using the :kconfig:option:`CONFIG_BT_SCAN_AND_INITIATE_IN_PARALLEL` Kconfig option.
+  * Support for the ``nrf54l15dk/nrf54l05/cpuapp`` and ``nrf54l15dk/nrf54l10/cpuapp`` board targets in the following samples:
+
+    * :ref:`direct_test_mode`
+    * :ref:`peripheral_hids_mouse`
+    * :ref:`peripheral_lbs`
+    * :ref:`power_profiling`
+    * :ref:`peripheral_uart`
+
+* Updated:
+
+  * Configurations of the following Bluetooth samples to make the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` the default settings backend for all board targets that use the MRAM technology:
+
+      * :ref:`bluetooth_central_hids`
+      * :ref:`peripheral_hids_keyboard`
+      * :ref:`peripheral_hids_mouse`
+      * :ref:`central_and_peripheral_hrs`
+      * :ref:`central_bas`
+      * :ref:`central_nfc_pairing`
+      * :ref:`central_uart`
+      * :ref:`peripheral_bms`
+      * :ref:`peripheral_cgms`
+      * :ref:`peripheral_cts_client`
+      * :ref:`peripheral_lbs`
+      * :ref:`peripheral_mds`
+      * :ref:`peripheral_nfc_pairing`
+      * :ref:`power_profiling`
+      * :ref:`peripheral_rscs`
+      * :ref:`peripheral_status`
+      * :ref:`peripheral_uart`
+      * :ref:`ble_rpc_host`
+
+    As a result, all :ref:`zephyr:nrf54h20dk_nrf54h20` configurations of the affected samples were migrated from the NVS settings backend to the ZMS settings backend.
+  * Testing steps in the :ref:`peripheral_hids_mouse` to provide the build configuration that is compatible with the `Bluetooth Low Energy app`_ testing tool.
+
+* :ref:`power_profiling` sample:
+
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+
+* :ref:`nrf_auraconfig` sample:
+
+  * Fixed the data sending (OCT-3251).
+    Data is now sent on all BISes when generated by the application (no SD card inserted).
+
+Bluetooth Fast Pair samples
+---------------------------
+
+* :ref:`fast_pair_locator_tag` sample:
+
+  * Updated the MCUboot bootloader configuration for the :ref:`zephyr:nrf54l15dk_nrf54l15` board target to enable the :kconfig:option:`CONFIG_FPROTECT` Kconfig option that is used to protect the bootloader partition against memory corruption.
+
+* :ref:`fast_pair_input_device` sample:
+
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
```

Comment:
Hi @peknis, please move this entry back to the changelog, it is relevant for 2.9-nRF54H20.